composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -210,6 +223,12 @@ private fun buildShardArguments(shardingOn: Boolean, shardIndex: Int, devices: I
     else -> emptyList()
 }
 
+private fun buildCoverageArguments(coverage: Boolean, coverageDir: String): List<Pair<String, String>> =
+    if (coverage) listOf(
+        "coverage" to "true",
+        "coverageFile" to "$coverageDir/coverage.ec"
```

Comment:
This needs to be `coverageFilePath` when running with testOrchestrator and then we will have to pull down .ec for each test case.